test/coder.encodeParam.js

```diff
@@ -70,16 +83,16 @@ describe('lib/solidity/coder', function () {
                                                                       '0000000000000000000000000000000000000000000000000000000000000005' + 
                                                                       '6761766f66796f726b0000000000000000000000000000000000000000000000'});
         test({ types: ['int', 'bytes'], values: [5, 'gavofyork'],
-                                                            expected: '0000000000000000000000000000000000000000000000000000000000000009' + 
-                                                                      '0000000000000000000000000000000000000000000000000000000000000005' + 
+                                                            expected: '0000000000000000000000000000000000000000000000000000000000000005' + 
+                                                                      '0000000000000000000000000000000000000000000000000000000000000009' + 
                                                                       '6761766f66796f726b0000000000000000000000000000000000000000000000'});
         test({ types: ['int', 'bytes', 'int', 'int', 'int', 'int[]'], values: [1, 'gavofyork', 2, 3, 4, [5, 6, 7]],
-                                                            expected: '0000000000000000000000000000000000000000000000000000000000000009' +
```

Comment:
why was this changed?
https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI see the sam / dave example